ruleset/rules/0755-office365_rules.xml

```diff
@@ -0,0 +1,928 @@
+        <options>no_full_log</options>
+        <group>MDATPAudit</group>
+    </rule>
+
+    <rule id="91607" level="5">
+        <if_sid>91532</if_sid>
+        <field name="office_365.RecordType" type="osregex">^82$</field>
+        <description>Office 365: Events generated when the file labeled with a sensitivity label is opened or renamed.</description>
+        <options>no_full_log</options>
+        <group>SensitivityLabelPolicyMatch</group>
+    </rule>
+
+    <rule id="91608" level="5">
+        <if_sid>91532</if_sid>
+        <field name="office_365.RecordType" type="osregex">^83$</field>
+        <description>Office 365: Event generated when sensitivity labels are applied, updated, or removed from a file.</description>
+        <options>no_full_log</options>
+        <group>SensitivityLabelAction</group>
+    </rule>
+
+    <rule id="91609" level="5">
+        <if_sid>91532</if_sid>
+        <field name="office_365.RecordType" type="osregex">^84$</field>
+        <description>Office 365: Events generated when a file labeled with a sensitivity label is opened or renamed.</description>
+        <options>no_full_log</options>
+        <group>SensitivityLabeledFileAction</group>
+    </rule>
+
+    <rule id="91610" level="3">
+        <if_sid>91532</if_sid>
+        <field name="office_365.RecordType" type="osregex">^85$</field>
+        <description>Office 365: Attack simulator events.</description>
+        <options>no_full_log</options>
+        <group>AttackSim</group>
+    </rule>
+
+    <rule id="91611" level="3">
+        <if_sid>91532</if_sid>
+        <field name="office_365.RecordType" type="osregex">^86$</field>
+        <description>Office 365: Events related to manual investigations in Automated investigation and response (AIR).</description>
+        <options>no_full_log</options>
+        <group>AirManualInvestigation</group>
+    </rule>
+
+    <rule id="91612" level="5">
+        <if_sid>91532</if_sid>
+        <field name="office_365.RecordType" type="osregex">^87$</field>
+        <description>Office 365: Security and compliance RBAC events.</description>
+        <options>no_full_log</options>
+        <group>SecurityComplianceRBAC</group>
+    </rule>
+
+    <rule id="91613" level="3">
+        <if_sid>91532</if_sid>
+        <field name="office_365.RecordType" type="osregex">^88$</field>
+        <description>Office 365: Attack simulator training events in Microsoft Defender for Office 365.</description>
+        <options>no_full_log</options>
+        <group>UserTraining</group>
+    </rule>
+
+    <rule id="91614" level="5">
+        <if_sid>91532</if_sid>
+        <field name="office_365.RecordType" type="osregex">^89$</field>
+        <description>Office 365: Events related to admin actions in Automated investigation and response (AIR).</description>
+        <options>no_full_log</options>
+        <group>AirAdminActionInvestigation</group>
+    </rule>
+
+    <rule id="91615" level="5">
```

Comment:
I think this level should be higher since these alerts come from Microsoft Defender.

Reply:
Done